Resources/Prototypes/_LostParadise/Catalog/Fills/Crates/weh_implant_crate.yml

```
@@ -85,3 +85,14 @@
     contents:
       - id: MacroBombImplanter
         amount: 5
+        
+- type: entity
+  id: LPPCrateWehImplants
+  name: Weh implant crate
+  description: Crate filled with 5 weh implants which let the user weh
+  parent: CrateMedical
+  components:
+  - type: StorageFill
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)